howso/utilities/utilities.py

```diff
@@ -1402,3 +1403,31 @@ def get_matrix_diff(matrix: pd.DataFrame) -> dict:
     differences_dict = {k: v for k, v in sorted(differences_dict.items(), key=lambda item: item[1], reverse=True)}
 
     return differences_dict
+
+
+def format_confusion_matrix(confusion_matrix: dict) -> t.Tuple[np.ndarray, list]:
```

Comment:
Realized that `tuple` has gotten the same treatment as `List` and `Dict`: https://docs.python.org/3/library/typing.html#typing.Tuple

```suggestion
def format_confusion_matrix(confusion_matrix: dict) -> tuple[np.ndarray, list]:
```